### ISSUE TITLE
Problem: Updating user replaces entire profile doc

### ISSUE THREAD

**Issue by samuelralak** [apacblocks/registration#43]
Problem: Updating user replaces entire profile doc. This deletes previously created attributes on user sign up.

Solution: Update each attribute individually. Instead of having the following
```javascript
return Meteor.users.update(userId, {
  $set: {
    username: userData.btcAddress,
    profile: userData
  }
});
```

update it to:
```javascript
return Meteor.users.update(userId, {
  $set: {
    username: userData.btcAddress,
    profile: {
      btcAddress: userData.btcAddress,
      telegram: userData.telegram,
      bio: userData.bio,
  }
});
```

**Comment by gazhayes** [comment]
Better fix this one asap @ioioio8888 and @samuelralak make sure to assign yourself immediately if you start working on this one, I will assign myself if I start doing it. Whoever gets time first.

**Comment by samuelralak** [comment]
on it

**Comment by samuelralak** [comment]
fixed by #47
